File: Exemplos SmartPOS/App_IntentDigitalHUB_Java/Elgin_IntentDigitalHUB_SmartPOS_Java-font/app/src/main/java/com/elgin/java_intentdigitalhub_smartpos/IntentDigitalHubService/IntentDigitalHubModule.java
package com.elgin.java_intentdigitalhub_smartpos.IntentDigitalHubService;

/**
 * Módulos necessário que são utilizados pelos comandos do IDH, para a ciração da intent com o filtro correto.
 */
public enum IntentDigitalHubModule {
    ELGINPAY("com.elgin.e1.digitalhub.ELGINPAY"),
    TERMICA("com.elgin.e1.digitalhub.TERMICA"),
    //O Scanner é o único módulo que não possuí comandos, apenas o seu módulo filtro é necessário para o ínicio da operação.
    SCANNER("com.elgin.e1.digitalhub.SCANNER");


    private final String intentPath;

    IntentDigitalHubModule(String intentPath) {
        this.intentPath = intentPath;
    }

    public String getIntentPath() {
        return this.intentPath;
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_SmartPOS_Kotlin-font/app/src/main/java/com/elgin/kotlin_intent_digital_hub_smartpos/IntentDigitalHubService/IntentDigitalHubModule.kt
package com.elgin.kotlin_intent_digital_hub_smartpos.IntentDigitalHubService

enum class IntentDigitalHubModule (val intentPath: String){
    ELGINPAY("com.elgin.e1.digitalhub.ELGINPAY"),
    TERMICA("com.elgin.e1.digitalhub.TERMICA"),
    //O Scanner é o único módulo que não possuí comandos, apenas o seu módulo filtro é necessário para o ínicio da operação.
    SCANNER("com.elgin.e1.digitalhub.SCANNER");
}